lambda/format-activity-log/format-activity-log.ts

```diff
@@ -1,79 +1,45 @@
-import { SQSEvent } from "aws-lambda";
+import { DynamoDBStreamEvent } from "aws-lambda";
+import { AttributeValue } from "@aws-sdk/client-dynamodb";
+import { unmarshall } from "@aws-sdk/util-dynamodb";
 
 import {
   SendMessageCommand,
   SendMessageRequest,
   SQSClient,
 } from "@aws-sdk/client-sqs";
-import {
-  Activity,
-  ActivityLogEntry,
-  TxmaEvent,
-  UserActivityLog,
-  UserData,
-} from "./models";
-
-const activityFromTxmaEvent = (txmaEvent: TxmaEvent): Activity => ({
-  type: txmaEvent.event_name,
-  client_id: txmaEvent.client_id,
-  timestamp: txmaEvent.timestamp,
-  event_id: txmaEvent.event_id,
-});
+import { ActivityLogEntry, allowedTxmaEvents, TxmaEvent } from "./models";
 
 const createNewActivityLogEntryFromTxmaEvent = (
   txmaEvent: TxmaEvent
 ): ActivityLogEntry => ({
+  event_id: txmaEvent.event_id,
   event_type: txmaEvent.event_name,
-  session_id: txmaEvent.user.session_id,
-  user_id: txmaEvent.user.user_id,
+  session_id: txmaEvent.session_id,
+  user_id: txmaEvent.user_id,
+  client_id: txmaEvent.client_id,
   timestamp: txmaEvent.timestamp,
-  activities: [activityFromTxmaEvent(txmaEvent)],
-  truncated: false,
+  reported_suspicious: false,
```

Comment:
Should this always be false?  Could we introduce a constant with a name that clarifies why this is always false.

Reply:
Yes default false at this stage, addressed by introduced a default const.